src/components/manage/Controlpanels/Groups/RenderGroups.jsx

```diff
@@ -105,9 +109,26 @@ class RenderGroups extends Component {
         <Table.Cell textAlign="right">
```

Comment:
Not related to this change but recently we are integrating our websites with Microsoft activedirectory and we have uuids as groupname. 
I propose we should have title instead of groupname (or both, like users) here :https://github.com/plone/volto/blob/d2c01244d3e764163c90eda26e14d00706c98550/src/components/manage/Controlpanels/Groups/RenderGroups.jsx#L84